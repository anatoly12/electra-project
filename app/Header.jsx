import React from 'react';

import styles from '../css/header.css';

class Header extends React.Component {

  render() {
    return (<div>Header</div>);
  }

};

export default Header;
