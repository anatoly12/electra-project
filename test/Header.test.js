import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../app/Header.jsx';

test('Header Test', () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
