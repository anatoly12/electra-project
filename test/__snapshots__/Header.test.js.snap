// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test 1`] = `
<div>
  Header
</div>
`;
